ingest/src/server_lora.rs

```diff
@@ -89,9 +89,11 @@ pub async fn grpc_server(shutdown: triggered::Listener, server_mode: String) ->
 
     // Initialize uploader
     let (file_upload_tx, file_upload_rx) = file_upload::message_channel();
-    let file_upload = file_upload::FileUpload::from_env(file_upload_rx).await?;
+    let file_upload =
+        file_upload::FileUpload::from_env_with_prefix("INGESTOR", file_upload_rx).await?;
 
-    let store_path = dotenv::var("INGEST_STORE")?;
+    let store_path =
+        std::env::var("INGEST_STORE").unwrap_or_else(|_| String::from("/var/data/ingestor"));
```

Comment:
```suggestion
       env::var("INGEST_STORE").unwrap_or_else(|_| String::from("/var/data/ingestor"));
```
you've got this imported in from a use statement above which looks like it can be combined with the other `std::{` on the line below it